Checks/PWR024/README.md

```diff
@@ -33,12 +33,13 @@ void example(int **A, int n, int m) {
 }
 ```
 
-This can be easily fixed by removing the `j` increment and calculating it as a
-function of variable `i`.
+This can be easily fixed by moving the iterator initializer inside the for
+statement.
 
 ```c
 void example(int **A, int n, int m) {
-  for (int i = 0; i < n; i++) {
+  int i;
+  for (i = 0; i < n; i++) {
```

Comment:
I definitely agree with @inaki-amatria. WDYT about including this explanation in the file? It would help users avoid this type of mistake when migrating loops to OpenMP's canonical form.

Reply:
I just wanted to make clear that OpenMP just requires the initialization, not the declaration, but if this causes any concern, I will revert this, no problem.

Thanks for the feedback!